apps/mosaico/src/Article/Article.purs

```diff
@@ -178,16 +178,23 @@ render imageComponent props =
             [ DOM.div
                 { className: "mosaico-article__authors-and-timestamps"
                 , children:
-                    [ foldMap
-                        (\authorName -> DOM.div
+                    map
+                        (\author -> DOM.div
                           { className: "mosaico-article__author"
                           , children: [ guard (article.articleType == Opinion) $
                                         renderOpinionType article.articleTypeDetails
-                                      , DOM.text authorName
+                                      , DOM.span_ [ DOM.text author.byline ]
+                                      , foldMap
+                                        (\authorEmail -> DOM.span
+                                                        {className: "mosaico-article__author-email"
+                                                        , children: [ DOM.text authorEmail ]
+                                                        }
+                                        )
+                                        author.email
                                       ]
                           })
-                        (_.byline <$> article.authors)
-                    , foldMap
+                        article.authors
+                    <> [foldMap
```

Comment:
How about changing this `foldMap [` to just `map`? I don't have much of preference either way but if it was changed one way on line 181 then let's use the same thing here.

Reply:
The reason it's foldMap is to prevent the span to exist if there is no email available (as we only show them for paper specific domains)

Reply:
Will leave it like this for now